snews_cs/cs_email.py

```diff
@@ -67,33 +101,21 @@ def send_feedback_mail(detector, attachment=None, message_content=None, given_co
         for contact in contacts:
             base_msg += f" {contact}"
             log.info(f"\t\t> Trying to send feedback to {contact} for {detector}")
-            out = _mail_sender([base_msg])
-            return out
+            _smtp_sender(message_content, subject, contact, attachment)
```

Comment:
I guess there are some redundant lines in this function that we might need to clean

Reply:
I believe in doing this, we could remove the for loop in favor of just allowing smtplib to send the various emails in the one smtp session with the server. (This would be more efficient)

Reply:
`smtp.sendmail()` accepts both a list and string, however the _smtp_sender helper function expects a string because  `email.mime.multipart.MIMEMultipart()` expects a single string instead of a list for constructing the email's envelope.  I suppose it's possible to handle that within the _smtp_sender function so that we could send a list.

My intent for _smtp_sender was to be a drop in replacement for the previous calls out to OS binaries, so that's how we got here. :-)